eventsourcing/src/test/java/org/axonframework/eventsourcing/configuration/AnnotatedEventSourcedEntityBuilderTest.java

```diff
@@ -30,79 +30,87 @@
 import org.axonframework.eventsourcing.annotation.reflection.EntityCreator;
 import org.axonframework.eventstreaming.EventCriteria;
 import org.axonframework.messaging.unitofwork.ProcessingContext;
+import org.axonframework.modelling.StateManager;
 import org.axonframework.modelling.repository.Repository;
+import org.axonframework.utils.StubLifecycleRegistry;
 import org.junit.jupiter.api.*;
 
 import java.lang.annotation.ElementType;
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
+import java.util.Set;
 
 import static org.junit.jupiter.api.Assertions.*;
 import static org.mockito.Mockito.*;
 
 /**
- * Test class validating the {@link AnnotatedEventSourcedEntityBuilder}.
+ * Test class validating the {@link AnnotatedEventSourcedEntityModule}.
  *
  * @author Mitchell Herrijgers
  * @author Steven van Beelen
  */
 class AnnotatedEventSourcedEntityBuilderTest {
```

Comment:
I think you should rename the class too.

Reply:
You are right, renaming!